src/common/src/lib.rs

```diff
@@ -59,3 +59,5 @@ pub mod test_prelude {
     pub use super::array::{DataChunkTestExt, StreamChunkTestExt};
     pub use super::catalog::test_utils::ColumnDescTestExt;
 }
+
+pub const RW_VERSION: &str = "v0.1.15";
```

Comment:
```suggestion
pub const RW_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Reply:
cargo version for debug version and the hard-code one for release mode

Reply:
Any motivation for differentiating them, if we don't want to include something like git revision? Before release, the version in `Cargo.toml` will be modified to `0.1.16`, so we'll also get `0.1.16` here.

Reply:
as discussed before, including git commit hash can be troublesome for compilation.
I thought we would keep `0.2.0-alpha` since it has been there for a long time.

> Before release, the version in Cargo.toml will be modified to 0.1.16, so we'll also get 0.1.16 here.
If so, there is no need for differentiating them. Will fix soon